src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -510,9 +524,8 @@ private String resolveContent(String result, Set<String> usedIds) {
             return Optional.of(EntryUtil.capitalizeFirst(entry.getType()));
         }
 
-        // TODO: Changed this to also consider alias fields, which is the expected
+        // Changed this to also consider alias fields, which is the expected
```

Comment:
You are sure that there are no unwanted side effects? ;-) (I've also looked at those lines quite a few times...)